src/main/java/net/snowflake/client/core/SFInputBase.java

```diff
@@ -0,0 +1,29 @@
+package net.snowflake.client.core;
+
+import java.util.Map;
+
+/**
+ * Base class for SF HTTP input types. Currently contains only the identical properties between
```

Comment:
Please add comment here , this is for Snowsight right now.

Reply:
I've added it to the `setAdditionalHttpHeaders()` method; I extracted this method to hold identical properties between the SF*Input classes based on your earlier comment, but it seemed unnecessary to refactor a large amount of the code in this PR